apps/admin/frontend/src/components/export_election_package_modal_button.tsx

```diff
@@ -108,17 +107,6 @@ export function ExportElectionPackageModalButton(): JSX.Element {
       }
       break;
 
-    case 'saving': {
-      actions = (
-        <Button onPress={closeModal} disabled>
```

Comment:
We're losing the disabled-while-saving state of cancel button here - should we be adding a conditional disable on line 92 when the mutation is running? (And additionally, a no-op in `closeModal()`?)

Reply:
+1 - was concerned enabling a 'cancel' button that doesn't actually cancel the action (and the fact that this state of the modal is step 1 of a 2-part flow, since there's a follow-up UI to eject the drive)